frontend/src/redux/actions/index.js

```diff
@@ -299,6 +303,27 @@ export const loadActionsFailed = message => ({
 
 // TODO more?
 
+export const fetchTimesOfDay = () => {
+    return dispatch => {
```

Comment:
Not worth it for this PR but Redux Toolkit has some very nice helper functions to avoid a lot of the boilerplate code here: https://redux-toolkit.js.org/usage/usage-guide#async-requests-with-createasyncthunk